samples/Samples.GraphQL/Samples.GraphQL.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <!-- override to remove net452 -->
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net461;netcoreapp2.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="GraphQL" Version="2.4.0" />
+    <PackageReference Include="GraphQL.StarWars" Version="1.0.0" />
```

Comment:
This package adds the GraphQL schema that this server responds to. I can copy the source inside the project if that would help for readability, but that lives here: https://github.com/graphql-dotnet/graphql-dotnet/tree/master/src/GraphQL.StarWars

Reply:
I might actually need to copy in some source anyways because this schema out of the box doesn't implement a Subscription interface (one of the 3 operations), but I am going to implement one so we can test against it.

Reply:
In the end I was able to just code against the GraphQL.StarWars package and I added my Subscription schema by injecting my own type in Startup.cs. If you would like to see the data objects copied into the sample project itself, I'd like to do that in a separate PR so I can check this in sooner.